test/Microsoft.TestPlatform.AcceptanceTests/AcceptanceTestBase.cs

```diff
@@ -109,4 +110,44 @@ public string GetDefaultRunSettings()
                                     </RunSettings>";
         return runSettingsXml;
     }
+
+    protected string GetIsolatedTestAsset(string assetName)
```

Comment:
Unrelated to this PR but fixes the failing acceptance tests that were blocking this PR.